src/components/Editor/HighlightLine.js

```diff
@@ -59,17 +96,18 @@ export class HighlightLine extends PureComponent<Props> {
     selectedFrame: Frame,
     selectedSource: SourceRecord
   ) {
-    if (!isDocumentReady(selectedSource, selectedLocation)) {
+    if (!this.shouldSetHighlightLine(selectedLocation, selectedSource)) {
       return;
     }
-
+    this.isInSteppingUpdate = false;
     const { sourceId, line } = selectedLocation;
+    const editorLine = toEditorLine(sourceId, line);
+    this.previousEditorLine = editorLine;
```

Comment:
should this be before the IF?

Reply:
Hmm, no according to my understanding, since the IF wants to look at the previous value of `this.previousEditorLine`.